### DISCUSSION TITLE
ID of the Conditional Element

### DISCUSSION THREAD

**Discussion by HitecExports** [slint-ui/slint#4837] [Q&A] [answered]
In the repetition syntax:
**for name[index] in model : id := Element { ... }**

And in Conditional Element syntax:
**if condition : id := Element { ... }**

there is assignment to **id** of the conditional element

Is it possible to use this **id** in the parent container element in the same manner as for usual "static" element?

Is it possible to call function of the conditional element with the help of it's **id** (as it possible to do with usual "static" element) ?

**Comment by ogoffart** [comment, marked as answer]
> Is it possible to use this id in the parent container element in the same manner as for usual "static" element?

No, this is not possible since the element might not exist.
